kivy/uix/screenmanager.py

```diff
@@ -980,8 +980,8 @@ def _screen_name_changed(self, screen, name):
 
     def add_widget(self, widget, *args, **kwargs):
         '''
-        .. versionchanged:: 2.1.0
-            Renamed argument `screen` to `widget`.
+        .. versionchanged:: 2.3.0
+            Raising exception for repeated screen name.
```

Comment:
Hi @matheussmoreira !

The old `versionchanged` should be kept, instead of editing it, you should consider adding a new one.

Reply:
Ok, I will retrieve the old one.